unit-10/delElementArray/main.js

```diff
@@ -0,0 +1,19 @@
+
+const delElementArray = () => {
+	let array = ['A', 'C', 'A', 'A', 'B', 'D', 'B', 'D']
+	let array2 = []
+
+	// let array = ['A', 'C', 'B', 'C', 'D', 'E', 'F', 'G']
+
+	for (let i = 0; i < array.length; i++) {
+		let array = ['A', 'C', 'A', 'A', 'B', 'D', 'B', 'D']
+		for (let j = i + 1; j <= array.length; j++) {
+			
+			if (array[i] === array[j]) {
+				array.splice(array[j],1)
```

Comment:
```suggestion
				array.splice(array[j], 1)
```
- Chú ý dấu phẩy
- Không được làm thay đổi array đầu vào